src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -93,19 +111,97 @@ const StreamGroup = createReactClass({
     SelectedGroupStore.toggleSelect(this.state.data.id);
   },
 
-  render() {
+  toggleShowLifetimeStats(showLifetimeStats) {
+    if (this.hoverWait) {
+      clearTimeout(this.hoverWait);
+    }
+
+    this.hoverWait = setTimeout(
+      () => this.setState({showLifetimeStats}),
+      MENU_CLOSE_DELAY
+    );
+
+    this.setState({showLifetimeStats});
+  },
+
+  getDiscoverUrl(filtered) {
+    const {organization, query, selection} = this.props;
     const {data} = this.state;
+
+    const {period, start, end} = selection.datetime || {};
+
+    const splitQuery =
+      filtered && query
+        ? query.match(/\S+:"[^"]*"?|\S+/g).filter(term => term.slice(0, 3) !== 'is:')
```

Comment:
You might be able to use utils/tokenizeSearch to parse and manipulate the query string here. Duplicating the expression pattern makes future changes around the search tokenization harder.